src/dash/admin_view/AdminDash.jsx

```diff
@@ -1,16 +1,168 @@
 import React from 'react';
+import ApproveTutors from './ApproveTutors.jsx';
+import TutorStatistics from './TutorStatistics.jsx';
+import Members from './Members.jsx';
 
-class DefaultDash extends React.Component {
+
+const tutors = [
+    {
+        first_name: 'Sruti',
+        last_name: 'Guhathakurta',
+        subjects: [{subject:'Math', start_grade:6, end_grade:12},{subject:'Science', start_grade:5, end_grade:7},{subject:'Computer Science', start_grade:6, end_grade:12}],
+        availability: {
+            Monday: [{start_time:'2:00', end_time:'3:00'}],
+            Tuesday: [{start_time:'6:00', end_time:'7:00'}],
+            Wednesday: [],
+            Thursday: [{start_time:'4:00', end_time:'5:00'}],
+            Friday: [],
+            Saturday: [],
+            Sunday: []
+        },
+        profile_picture: '/images/default_user_img.png',
+        bio: 'Hello, my name is Sruti.',
+        email: 'sruti@gatech.edu',
+        class_standing: 'Senior',
+        rating: 5,
+        online: false,
+        gender: 'female',
+        approved:true
```

Comment:
Is approved a boolean or string? I remember seeing in our db code that we're treating it like a string / enum. 

I think the possible states for approval are:
- New
- In Review
- Approved

Or do you think we only need 2 states?

Reply:
I think I had it as a boolean here, but it can be changed pretty easily! I don't think I did anything with approval in my code. What would differentiate a tutor who's new and in review?

Reply:
Not sure tbh, I think that was something @ganhari123 mentioned. Any thoughts @ganhari123 ?